modules/sov-ibc-transfer/src/context.rs

```diff
@@ -72,6 +86,46 @@ impl<'ws, S: Spec> IbcTransferContext<'ws, S> {
             })
     }
 
+    fn create_token(
+        &self,
+        token_name: String,
+        minter_address: S::Address,
+        authorized_minters: Vec<S::Address>,
+    ) -> Result<TokenId, TokenTransferError> {
```

Comment:
can you please add some comments here? Also, we may use `ibc_denom_trace` instead of `token_name` here as we will only pass an IBC denom trace.